docs/releases.rst

```diff
@@ -1,6 +1,333 @@
+As a user you can now go back to your profile page where you can change your password or spam settings. And then go back to Roundcube again.
+
+PROXY PROTOCOL Support
+^^^^^^^^^^^^^^^^^^^^^^
+
+Reverse proxies can connect to Mailu with the `proxy protocol <https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt>`_ for HTTP and Mail. Below is a small example for Traefik connecting via proxy protocol to Mailu
+
+.. code-block:: bash
+
+  # Static configuration
+  providers:
+  file:
+    directory: "/opt/traefik/conf"
+
+  entryPoints:
+    mailu-web:
+      # Listen on port 8081 for incoming requests
+      address: :443
+    mailu-smtp:
+      address: :25
+    mailu-imaps:
+      address: :993
+    mailu-smtps:
+      address: :465
+    mailu-starttls:
+      address: :587
+
+  # From dynamic configuration /opt/traefik/conf
+  tls:
+    certificates:
+      - certFile: /etc/letsencrypt/live/mydomain.com/fullchain.pem
+        keyFile: /etc/letsencrypt/live/mydomain.com/privkey.pem
+
+  tcp:
+    routers:
+      mailu-web:
+        entryPoints:
+          - mailu-web
+        rule: "HostSNI(`*`)"
+        service: "mailu-web"
+      mailu-smtp:
+        entryPoints:
+          - mailu-smtp
+        rule: "HostSNI(`*`)"
+        service: "mailu-smtp"
+      mailu-imaps:
+        entryPoints:
+          - mailu-imaps
+        rule: "HostSNI(`*`)"
+        service: "mailu-imaps"
+      mailu-smtps:
+        entryPoints:
+          - mailu-smtps
+        rule: "HostSNI(`*`)"
+        service: "mailu-smtps"
+      mailu-starttls:
+        entryPoints:
+          - mailu-starttls
+        rule: "HostSNI(`*`)"
+        service: "mailu-starttls"
+    services:
+      mailu-web:
+        loadBalancer:
+          proxyProtocol:
+            version: 2
+          servers:
+            - address: "MailuServer:443"
+      mailu-smtp:
+        loadBalancer:
+          proxyProtocol:
+            version: 2
+          servers:
+            - address: "MailuServer:25"
+      mailu-smtps:
+        loadBalancer:
+          proxyProtocol:
+            version: 2
+          servers:
+            - address: "MailuServer:465"
+      mailu-starttls:
+        loadBalancer:
+          proxyProtocol:
+            version: 2
+          servers:
+            - address: "MailuServer:587"
+      mailu-imaps:
+        loadBalancer:
+          proxyProtocol:
+            version: 2
+          servers:
+            - address: "MailuServer:993"
+
+Security hardening
+^^^^^^^^^^^^^^^^^^
+
+We have gone further than ever. Now Mailu containers drop their privileges and communicate on separate networks. They also share the same base image where on x86 `a Hardened memory allocator <https://github.com/GrapheneOS/hardened_malloc>`_ is configured.
+
+Webmails which are running PHP make use of `Snuffleupagus <https://github.com/jvoisin/snuffleupagus>`_.
+
+
+New Functionality & Improvements
```

Comment:
It might be good to add that the current Mailu Helm Chart is not compatible, and that they need to wait until version `1.0.0` is released (we'll do with @micw as soon as Mailu is released and containers are available)